CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+## [1.0-beta-1] - 2018-12-XX
+### Added
```

Comment:
Maybe an overkill to add subsections for now

Reply:
Removed!